packages/datagateway-download/src/downloadApiHooks.test.tsx

```diff
@@ -542,4 +739,762 @@ describe('Download Cart API react-query hooks test', () => {
+          transport: 'https',
+          userName: 'test user',
+        },
+        {
+          createdAt: '2020-02-28T15:57:28Z',
+          downloadItems: [{ entityId: 4, entityType: 'investigation', id: 4 }],
+          email: 'test4@email.com',
+          facilityName: 'LILS',
+          fileName: 'test-file-4',
+          fullName: 'Person 4',
+          id: 4,
+          isDeleted: 'No',
+          isEmailSent: true,
+          isTwoLevel: false,
+          preparedId: 'test-prepared-id',
+          sessionId: 'test-session-id',
+          size: 4000,
+          status: 'downloadStatus.expired',
+          transport: 'globus',
+          userName: 'test user',
+        },
+        {
+          createdAt: '2020-03-01T15:57:28Z[UTC]',
+          downloadItems: [{ entityId: 5, entityType: 'investigation', id: 5 }],
+          email: 'test5@email.com',
+          facilityName: 'LILS',
+          fileName: 'test-file-5',
+          fullName: 'Person 5',
+          id: 5,
+          isDeleted: 'No',
+          isEmailSent: true,
+          isTwoLevel: false,
+          preparedId: 'test-prepared-id',
+          sessionId: 'test-session-id',
+          size: 5000,
+          status: 'downloadStatus.paused',
+          transport: 'globus',
+          userName: 'test user',
+        },
+      ];
+
+      axios.get = jest.fn().mockResolvedValue({ data: mockDownloadItems });
+
+      const { result, waitFor } = renderHook(() => useDownloads(), {
+        wrapper: createReactQueryWrapper(),
+      });
+
+      await waitFor(() => result.current.isSuccess);
+
+      expect(axios.get).toHaveBeenCalledWith(
+        `${mockedSettings.downloadApiUrl}/user/downloads`,
+        {
+          params: {
+            sessionId: null,
+            facilityName: mockedSettings.facilityName,
+            queryOffset: 'where download.isDeleted = false',
+          },
+        }
+      );
+      expect(result.current.data).toEqual(mockFormattedDownloadItems);
+    });
+
+    it('should call handleICATError on failure', async () => {
+      axios.get = jest.fn().mockRejectedValue({
+        message: 'Test error message',
+      });
+
+      const { result, waitFor } = renderHook(() => useDownloads(), {
+        wrapper: createReactQueryWrapper(),
+      });
+
+      await waitFor(() => result.current.isError);
+
+      expect(axios.get).toHaveBeenCalledWith(
+        `${mockedSettings.downloadApiUrl}/user/downloads`,
+        {
+          params: {
+            sessionId: null,
+            facilityName: mockedSettings.facilityName,
+            queryOffset: 'where download.isDeleted = false',
+          },
+        }
+      );
+      expect(handleICATError).toHaveBeenCalledWith({
+        message: 'Test error message',
+      });
+    });
+  });
+
+  describe('useDownloadDeleted', () => {
+    it('should delete download with given id and update the download list upon success', async () => {
+      const mockDownloadItems: Download[] = [
```

Comment:
Again, couldn't this just reference the global `mockDownloadItems` like `[mockDownloadItems[0]]` to get a single download item mock?